spec/spec_helper.rb

```diff
@@ -13,6 +13,8 @@
 DUMMY_CLOUD = "test123"
 TEST_IMAGE_URL = "http://cloudinary.com/images/old_logo.png"
 TEST_IMG = "spec/logo.png"
+TEST_VIDEO = "spec/sample.mp4"
+TEST_RAW = "spec/sample.txt"
```

Comment:
Mind that `TEST_IMG` is a real file in `spec` folder, while `spec/sample.txt` does not exist, there is `spec/docx.docx` file that can be used as raw.